crypto/asn1/a_int.c

```diff
@@ -229,12 +229,10 @@ static size_t asn1_put_uint64(unsigned char b[sizeof(uint64_t)], uint64_t r)
 }
 
 /*
- * Absolute value of INT64_MIN: we can't just use -INT64_MIN as it produces
+ * Absolute value of INT64_MIN: we can't just use -INT64_MIN as gcc produces
  * overflow warnings.
  */
-
-#define ABS_INT64_MIN \
-    ((uint64_t)INT64_MAX + (uint64_t)(-(INT64_MIN + INT64_MAX)))
+#define ABS_INT64_MIN ((uint64_t)INT64_MAX + (-(INT64_MIN + INT64_MAX)))
```

Comment:
It was in x_int64.c, not here. While I commented that it should be +1 there *and* here, I've chosen to keep it here. Because x_int64.c didn't bother with corner cases from start, while this module kind of did. So I figured that it wouldn't be inappropriate to keep it here after all...